settings.gradle.kts

```diff
@@ -14,3 +14,17 @@ dependencyResolutionManagement {
         }
     }
 }
+
+val isCiServer = System.getenv().containsKey("CIRRUS_CI")
+val isMasterBranch = System.getenv()["CIRRUS_BRANCH"] == "master"
+val buildCacheHost: String = System.getenv().getOrDefault("CIRRUS_HTTP_CACHE_HOST", "localhost:12321")
+buildCache {
```

Comment:
Probably, yes. I implemented here [the recommendations from CirrusCI](https://cirrus-ci.org/guide/writing-tasks/#http-cache) in order to improve the build time. This can be removed or updated.

Reply:
Perfect!